cli/src/main/kotlin/OrtServerMain.kt

```diff
@@ -65,18 +67,36 @@ class OrtServerMain : SuspendingNoOpCliktCommand(COMMAND_NAME) {
 
         subcommands(AuthCommand(), RunsCommand())
 
+        @Suppress("ForbiddenMethodCall")
```

Comment:
Maybe add a comment why it's ok to not use `org.eclipse.apoapsis.ortserver.utils.logging.runBlocking` here?

Reply:
The CLI does not have a dependency to the utils package, and at this point I don't want to introduce it (due to #2031).
Maybe we should introduce an exception for the `ForbiddenMethodCall` for the CLI?

Reply:
> Maybe we should introduce an exception for the `ForbiddenMethodCall` for the CLI?

Sounds reasonable to me, if it can be done with detekt.

Reply:
Looks like the [generic `excludes` mechanism](https://detekt.dev/docs/introduction/configurations#path-filters--excludes--includes) could be used for this.